### PR TITLE
Update tests for v0.3

### PR DESCRIPTION
**Pull request by shinich1**
Before submitting, please check the following:

- Make sure you have tests for the new code and that test passes (run `nox`)
- If applicable, add a line to the [unreleased] part of CHANGELOG.md, following [keep-a-changelog](https://keepachangelog.com/en/1.0.0/).
- Format added code by `ruff`
  - See `CONTRIBUTING.md` for more details
- Make sure the checks (github actions) pass.
- Check that the docs compile without errors (run `make html` in `./docs/` - you may need to install dependency for sphinx docs, see `docs/requirements.txt`.)

Then, please fill in below:

**Context (if applicable):**

Fix tests to account for the deprecations.

**Description of the change:**

- fixed `test_pattern`, `test_transpiler`. rename `pattern.standardize`, `pattern.shift_signals` argument names.
- removed `CommandNode` class which should also be deprecated.

**Related issue:**

### CODE REVIEW

**Comment by EarlMilktea** [comment]
@shinich1 I will directly push some changes to this branch for easier review. Watch out!

File: graphix/pattern.py [hunk 1]
```diff
@@ -270,23 +269,16 @@ def standardize(self, method="direct"):
             'direct' fast standardization implemented as `standardize_direct()`
             defaults to 'direct'
         """
-        if method == "direct":
+        if method == "direct":  # faster implementation
```
Comment (EarlMilktea) [inline graphix/pattern.py:272]: What is the purpose to maintain two methods (`direct` and `mc`)?

**Review by EarlMilktea — COMMENTED** [review]
MEMO: Doc CI is failing. Could you fix it?

File: graphix/pattern.py (hunk 1 above)
Comment (shinich1) [reply graphix/pattern.py:272]: Indeed we should remove `mc` if others agree. cc @thierry-martinez @masa10-f

Comment (EarlMilktea) [reply graphix/pattern.py:272]: I see. So I will assume `mc` can be dropped when refactoring `Pattern`.

Comment (shinich1) [reply graphix/pattern.py:272]: 1a759dabbb5eb224defa2a76ead7762832c652b6 another branch for deprecating `mc` method

**Review by thierry-martinez — APPROVED** [review]
LGTM! Thanks, and sorry for the delay.

**Pull request merged**